modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -227,8 +228,8 @@ private void verifyApplicationName(String name) {
       if (c == 0) {
         reason = "null character not allowed @" + i;
         break;
-      } else if (c == '/' || c == '.') {
-        reason = "invalid character @" + i;
+      } else if (c == '/' || c == '.' || c == ':') {
```

Comment:
I found this but not sure if its definitive: http://hadoop.apache.org/docs/r2.5.2/hadoop-project-dist/hadoop-common/filesystem/model.html